### ISSUE TITLE
Use yarn or npm but not the 2

### ISSUE THREAD

**Issue by iorlandini** [openunite/openunite#29]
I propose to choose one of the 2 and remove the `yarn.lock` file or `package-lock.json`

**Comment by luisfarzati** [comment]
If we need to make this decision, then I vote for npm. Despite my love for yarn, when it comes to OSS my stake is on using the most de facto tool, so => npm.

**Comment by alanreid** [comment]
Yeah, I was thinking the same today, it's a mess. Let's stick with npm.

**Comment by alanreid** [comment]
Note: We'd need to update our Dockerfiles, docs, etc
